src/docs/forms/components/TextInput/examples/02-SimpleTextInput.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import Form, {
+    TextInput
+} from 'doj-react-adminlte/Form';
+
+export default class SimpleTextInput2 extends React.Component {
```

Comment:
Please rename your classes to reflect what the example does. For example, this particular class can be renamed to `PasswordInput`. Consequently, change the file name of the example to `02-PasswordInput.js`. Do this with other examples as well.